examples/tutorials/1_generating_signals.ipynb

```diff
@@ -0,0 +1,359 @@
+    "as the detector has different sensitivities depending on its orientation.\n",
+    "\n",
+    "The frequency is modulated in a quasi-periodic fashion due to the translation of the interferometric\n",
+    "detector around the Sun. The downwards trend is due to the presence of a negative source spindown due to the\n",
+    "emission of energy as gravitational waves.\n",
+    "\n",
+    "The instantaneous frequency of a CW at the detector can be expressed as\n",
+    "$$\n",
+    "f(t) = \\left[f_0 + f_1 \\left(t - t_{\\text{ref}}\\right)\\right]\\left[1 + \\frac{\\vec{v} \\cdot \\hat{n}}{c}\\right],\n",
+    "$$\n",
+    "where $\\vec{v}/c$ is the velocity of the detector normalized by the speed of light in vaccum \n",
+    "and $t_{\\text{ref}}$ is a fiducial reference time at which $f_0$ and $f_1$ are specified.\n",
+    "\n",
+    "Detector velocities can be retrieved from SFT files using `pyfstat.DetectorStates`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1d9aa035",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "states = pyfstat.DetectorStates().get_multi_detector_states_from_sfts(writer.sftfilepath,\n",
+    "                                                                      central_frequency=writer.F0,\n",
+    "                                                                      time_offset=0)\n",
+    "\n",
+    "ts = np.array([data.tGPS.gpsSeconds for data in states.data[0].data])\n",
+    "velocities = np.vstack([data.vDetector for data in states.data[0].data]).T\n",
+    "\n",
+    "n = np.array([[\n",
+    "    np.cos(writer.Alpha) * np.cos(writer.Delta), # Cartesian X\n",
+    "    np.sin(writer.Alpha) * np.cos(writer.Delta), # Cartesian Y\n",
+    "    np.sin(writer.Delta)                         # Cartesian Z\n",
+    "]])\n",
+    "\n",
+    "f_inst = (\n",
+    "    (writer.F0 + (ts - writer.tref) * writer.F1)\n",
+    "    * (1 + np.dot(n, velocities))\n",
+    ")[0]\n",
+    "    \n",
+    "for a in ax:\n",
+    "    for off in [-2, 2]:\n",
+    "        a.plot((ts - ts[0])/1800, f_inst + off / writer.Tsft,\n",
+    "               color=\"white\", ls=\"--\", label=\"Instantaneous frequency\" if off == 2 else \"\")\n",
+    "    a.legend()\n",
+    "fig"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0491de7c",
+   "metadata": {},
+   "source": [
+    "The presence of visible CW features in the spectrogram can be quantified using the so-called\n",
+    "[sensitivity depth](https://arxiv.org/abs/1808.02459), which is defined as the ratio between the noise\n",
+    "and CW amplitude\n",
+    "$$\n",
+    "\\mathcal{D} = \\frac{\\sqrt{S_{\\text{n}}}}{h_0}\\;.\n",
+    "$$\n",
+    "\n",
+    "Current searches for CW signals from unknown sources are able to detect a sensitivity depth ranging between\n",
+    "$10\\;\\text{Hz}^{-1/2}$ and $50\\;\\text{Hz}^{-1/2}$. Visual features, however, tend to disappear around a depth\n",
+    "of $20\\;\\text{Hz}^{-1/2}$."
```

Comment:
This sounds too optimistic for visual features according to my intuition, but I have problems running locally right now, will think about / test it again later.